tests/test_integration.py

```diff
@@ -90,32 +76,18 @@ def test_load_remote_device_with_backend_instance(self, d, backend):
         except QiskitBackendNotFoundError:
             pytest.skip("Backend is not compatible with specified device")
 
-        dev = qml.device("qiskit.remote", wires=2, backend=backend_instance, shots=1024)
-        assert dev.num_wires == 2
-        assert dev.shots == 1024
-        assert dev.short_name == "qiskit.remote"
-        assert dev.provider is None
-        assert dev.capabilities()["returns_state"] == (backend in state_backends)
-
-    @pytest.mark.parametrize("d", pldevices)
-    def test_load_remote_device_by_name(self, d, backend):
-        """Test that the qiskit.remote device loads correctly when passed a provider and a backend
-        name. This test is equivalent to `test_load_device` but on the qiskit.remote device instead
-        of specialized devices that expose more configuration options."""
-
-        # check compatibility between provider and backend, and skip if incompatible
-        is_compatible, failure_msg = check_provider_backend_compatibility(d, backend)
-        if not is_compatible:
-            pytest.skip(failure_msg)
-
-        _, provider = d
+        if backend_instance.configuration().n_qubits is None:
+            pytest.skip("No qubits?")
```

Comment:
When does this happen? If its simulators with an arbitrary number of qubits, could we do 

```
n_qubits = backend_instance.configuration().n_qubits
n_wires = n_qubits if n_qubits else 10
```
and then pass `n_wires` below to avoid skipping those devices in tests?

Reply:
This was probably added when I was experimenting with adding FakeManila to the pl_devices. I probably was in the wrong branch lol. I deleted it from this branch (since it's not relevant) but something similar might come back when addressing using FakeManila for the integration tests.